state-chain/pallets/cf-lp/src/lib.rs

```diff
@@ -360,4 +364,16 @@ impl<T: Config> LpBalanceApi for Pallet<T> {
 		});
 		Ok(())
 	}
+
+	fn asset_balances(who: &Self::AccountId) -> Vec<AssetBalance> {
```

Comment:
I think it probably makes sense to delete this AssetBalance type, and just use a tuple of ChainAsset, and AssetBalance.

Reply:
Basically so it matches the interface of: cf_asset_balances

Reply:
I think we can't because it's returned from the lp-api, and I'm pretty sure we don't want to have breaking changes, right?

Reply:
There are two types both called AssetBalance, I mean the other one.

Reply:
But yes we don't want breaking changes.